minerl/herobraine/wrappers/downscale_wrapper.py

```diff
@@ -0,0 +1,28 @@
+from copy import deepcopy
+
+import cv2
+import numpy as np
+from gym import ObservationWrapper
+
+from minerl.herobraine.hero.spaces import Box, Dict
+
+
+class DownscaleWrapper(ObservationWrapper):
+    def __init__(self, env, downscale_res=(64, 64)):
+        super().__init__(env)
+        self.downscale_res = downscale_res
+
+        obs_space = env.observation_space
+
+        obs_dict = {key: obs_space[key] for key in obs_space}
+        # This is specifically tailored for the MineRL observations...
+        obs_dict['pov'] = Box(0, 255, downscale_res + (3,),
+                              dtype=np.uint8)
+        self.observation_space = Dict(obs_dict)
+
+    def observation(self, obs):
```

Comment:
- [ ] Figure out why we are getting the wrong dimensions (either at this wrapper layer or at the EnvSpec HighRes wrapper layer.